### PR TITLE
Added a Contact Us screen with sheets API integration

### LINKED ISSUE #70: [FEATURE] Creating a Contact-Us screen in the application
### Description

I'd create a contact-us screen, where the users would be presented with a form to fill in, and the responses would be sent to a secured Google sheet, where all responses would be listed down, along with user's email and timestamp to differentiate between the entries. I'd do the integration of google sheets with the app via Sheets API. 

### Screenshots
h

### Additional information

Google sheets api: https://developers.google.com/sheets/api/guides/concepts

### PR DESCRIPTION
**Pull request by jaivsh**
## Description

I've implemented a user feedback and query system through Google sheets API. This consists of a ```contact us``` page through which the users can fill in their queries through the implemented form UI. Then, this query is published via sheets API and it shows up in the alloted google sheet. 
Screenshot_20230801_161752
Screenshot 2023-08-01 at 4 27 43 PM

Link to the Google sheet: https://docs.google.com/spreadsheets/d/1UVyKcz7icc0LCvwvILNKfZzwxmaw8MvUbqTyzhgXehI/edit?usp=sharing
<br>
I've implemented and deployed the following method via script extension in the sheet to enable API:
<br>
Screenshot 2023-08-01 at 4 29 13 PM

Fixes #70 

## Steps to transfer ownership of sheets

This API has been created by using the scripts feature of google sheets, where i created the relevant functions and deployed them for use. The sheet is open for anyone currently. The steps for making it private to Friday application owners are as follows:

- Open the sheet in the owner's Google account.
- Then, open the scripts module from Extension -> App Script.
- Create new deployment as Web-App.
- In execute as, select the relevant email address, and in "Who has access", select "Myself". This would restrict access to app's owners. The sheet can then be opened and edited by apps owners only.

### CODE REVIEW

**Comment by jaivsh** [comment]
@bhattabhi013 please reiew this PR asap!

File: lib/Settings/contact.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  @override
+  State<ContactScreen> createState() => _ContactScreenState();
+}
+
+class _ContactScreenState extends State<ContactScreen> {
+
+  TextEditingController controller = TextEditingController();
+  TextEditingController controller1 = TextEditingController();
+  TextEditingController controller2 = TextEditingController();
```
Comment (bhattabhi013) [inline lib/Settings/contact.dart:17]: Change the names of the controllers. ex: `emailController`

**Review by bhattabhi013 — CHANGES_REQUESTED** [review]

File: lib/Settings/contact.dart (hunk 1 above)
Comment (jaivsh) [reply lib/Settings/contact.dart]: @bhattabhi013 changed accordingly

**Comment by jaivsh** [comment]
@bhattabhi013 please check, I've changed the contact.dart accordingly

**Comment by bhattabhi013** [comment]
Please resolve the conflicts

**Comment by jaivsh** [comment]
@bhattabhi013 resolved conflicts, please check

**Review by bhattabhi013 — APPROVED** [review]
LGTM 🚀

**Comment by jaivsh** [comment]
Hey @bhattabhi013 please give a level 3 tag here, there is no tag with this PR

**Pull request merged**